doc/changes/names.inc

```diff
@@ -425,3 +425,5 @@
 .. _Senwen Deng: https://snwn.de
 
 .. _Michiru Kaneda: https://github.com/rcmdnk
+
+.. _Scott Huberty:  https://orcid.org/0000-0003-2637-031X
```

Comment:
missing empty line at the end of the file

Reply:
> can you touch a test in test_egi.py file eg to make sure the data in the parsed properly using the files we already use for testing? you could test that utc_offset key is present now for example.

Hi @agramfort yes I tested my revised code on your test_egi files before I pushed my changes. I  confirmed that the `meas_date` and `utc-offset` returned from my revised code match the timestamp in the `.mff/info.xml` file provided by EGI. I also ran the test functions in `test_egi.py` that I could (some required `mffpy` which  I don't have). You can see the output for my tests in the attached file
[mydebug_tests.html.zip](https://github.com/mne-tools/mne-python/files/8154870/mydebug_tests.html.zip)
.

Scott

Reply:
> missing empty line at the end of the file

I see - would you like me to add the empty line to names.inc and push to my fork?